package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "build": "nx affected:build",
     "change": "beachball change",
+    "check-dependencies": "syncpack list -pP",
```

Comment:
There is a list of script an engineer is supposed to run before pushing - lint, type-check, test... now this one.
What is the best way to know about that list? Not a big fan of precommit hook but can we have something like `ci` of `verify` which I can run (or to know the complete list)?